src/sknnr_spatial/types.py

```diff
@@ -1,11 +1,12 @@
 from __future__ import annotations
 
 from collections.abc import Sequence
-from typing import Any, TypeVar, Union
+from typing import Union
```

Comment:
From what I understand, `typing_extensions` does support `Union`, but it's a pure alias of `typing.Union`.  Is the idea to rely as little as possible on `typing_extensions` and use classes from `typing` when the `typing_extensions` version is just an alias?